CHANGELOG.md

```diff
@@ -14,6 +14,9 @@ and what APIs have changed, if applicable.
 
 ## [Unreleased]
 
+## [29.58.7] - 2024-09-13
+Add WildcardResourceSubscriber for restli-resource-explorer subscribe to all resources
```

Comment:
Nit: should there be a `-` at the beginning of this line? 

Also, I don't think we necessarily need to mention `restli-resource-explorer` specifically here, any user could use `WildcardResourceSubscriber` to subscribe to all resources, right? Plus then we don't need to name an internal service

Reply:
Yeah. Thanks for suggestions. Update